chapter-03/1_4_testing/pocketlog/logger_test.go

```diff
@@ -2,6 +2,7 @@ package pocketlog_test
 
 import "github.com/ablqk/tiny-go-projects/chapter-03/1_4_testing/pocketlog"
 
+// ExampleLogger_Debug should fail, because the code is coming in the next chapter.
```

Comment:
Is that really what we want?

Reply:
not a very clear message. I'd recommend discarding the func.